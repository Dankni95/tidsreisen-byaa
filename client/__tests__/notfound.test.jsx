import React from "react";
import { MemoryRouter } from "react-router-dom";
import { NotFound } from "../src/components/NotFound.jsx";
import renderer from 'react-test-renderer';

//TODO: Simulere click (se linje 25 i NotFound.jsx)

describe("NotFound", () => {
    it("shows snapshot", async () => {
        const component = renderer.create(
            <MemoryRouter>
                <NotFound />
            </MemoryRouter>
        );
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot();
    });
});