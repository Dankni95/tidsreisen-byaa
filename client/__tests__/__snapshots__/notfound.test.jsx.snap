// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound shows snapshot 1`] = `
<div
  id="background-modal-notfound"
/>
`;
